src/util-napatech.c

```diff
@@ -916,16 +926,28 @@ int NapatechGetStreamConfig(NapatechStreamConfig stream_config[])
 
                     char copystr[16];
                     strlcpy(copystr, stream->val, 16);
-                    ByteExtractStringUint16(&start, 10,  0, copystr);
-                    ByteExtractStringUint16(&end, 10,  0, strchr(copystr, '-') + 1);
+                    if (StringParseUint8(&start, 10, 0, (const char *)copystr) < 0) {
```

Comment:
Why from `ByteExtractStringUint16` to `StringParseUint8` ?

Reply:
Because for all configuration vars, we want to make sure that the value is checked strictly i.e. not followed by any extra chars than required

Reply:
I meant why `8` instead of `16` ?

Reply:
This seems to be a mistake which should error out during compilation. Maybe I don't get it because of missing napatech support. Fixing. Thanks.